ssg/controls.py

```diff
@@ -349,7 +349,11 @@ def load(self):
         self.title = ssg.utils.required_key(yaml_contents, "title")
         self.source = yaml_contents.get("source", "")
         self.reference_type = yaml_contents.get("reference_type", None)
-        self.product = yaml_contents.get("product", None)
+        yaml_product = yaml_contents.get("product", None)
+        if type(yaml_product) is list:
```

Comment:
using isinstance() is preffered over type()

Reply:
Thanks changed it in [83eaa21](https://github.com/ComplianceAsCode/content/pull/12454/commits/83eaa2100cbeb50ce03771cea2eb09dd505c7a87)